mlir/utils/performance/perfRunner.py

```diff
@@ -1048,50 +1117,7 @@ def findRunCommand(filename):
     print("WARNING: cannot find valid RUN command in ", filename)
     return None, None
 
-# Extract testVector and test function name from the test file
-def getFusionTestInfo(filename, paths: Paths):
```

Comment:
I had missed getFusionTestInfo when I updated everything to use runPipeline.  When I started updating it, I noticed that everything up to the tuningKey process was identical with runFusionKernel, so I abstracted the common part into makeBasicFusionPipeline.

Then I realised that it was doing redundant work, because we'd collect tests and call getFusionTestInfo on them, then loop through the collected tests and call runFusionKernel on those, and that would do the basic pipeline twice.  I recast it to do the basics once and save the mlir, and merged the collection and running loops so it doesn't have to save the mlir for very long.  More notes below.